pkg/detectors/buildkite/v1/buildkite.go

```diff
@@ -52,17 +52,10 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 		}
 
 		if verify {
-			req, err := http.NewRequestWithContext(ctx, "GET", "https://api.buildkite.com/v2/access-token", nil)
-			if err != nil {
-				continue
-			}
-			req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", resMatch))
-			res, err := client.Do(req)
-			if err == nil {
-				defer res.Body.Close()
-				if res.StatusCode >= 200 && res.StatusCode < 300 {
-					s1.Verified = true
-				}
+			isVerified, verificationErr := VerifyBuildKite(ctx, client, resMatch)
+			s1.Verified = isVerified
+			if verificationErr != nil {
```

Comment:
This check already happens in `SetVerificationError`.

https://github.com/kashifkhan0771/trufflehog/blob/a6e2b9995d5e6f9db9d7dc3430fcfed04f6688fb/pkg/detectors/detectors.go#L115-L116